config/example_options.ini

```diff
@@ -144,6 +144,9 @@ ShowConfigOnLaunch = no
 # do not need to use "skip f" to force-skip a song, they will instead force-skip by default.
 LegacySkip = no
 
+# Leave servers if the owner is not found in them.
+LeaveNonOwners = no
```

Comment:
Maybe we can come up with a more descriptive name for this config option?

Reply:
I could understand that I just didn't want to make it to long, and keep it the same as the other config options. Will think of a better name.

Reply:
How does LeaveNonBotOwners sound to you?